easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.3.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,165 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.3.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+builddependencies = [
+    ('Bazel', '3.6.0'),
+    ('protobuf', '3.10.0'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.23.0', '-nodocs'),
+    ('pybind11', '2.4.3', versionsuffix),
+]
+dependencies = [
+    ('Python', '3.8.2'),
+    ('h5py', '2.10.0', versionsuffix),
+    ('cURL', '7.69.1'),
+    ('double-conversion', '3.1.5'),
+    ('flatbuffers', '1.12.0'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '1.11.12'),
+    ('ICU', '66.1'),
+    ('JsonCpp', '1.9.3'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('LMDB', '0.9.24'),
+    ('NASM', '2.14.02'),
+    ('nsync', '1.24.0'),
+    ('SQLite', '3.31.1'),
+    ('PCRE', '8.44'),
+    ('protobuf-python', '3.10.0', versionsuffix),
+    ('libpng', '1.6.37'),
+    ('snappy', '1.1.8'),
+    ('SWIG', '4.0.1'),
+    ('zlib', '1.2.11'),
+]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
+use_pip = True
+
+# Dependencies created and updated using findPythonDeps.sh:
+# https://gist.github.com/Flamefire/49426e502cd8983757bd01a08a10ae0d
+exts_list = [
+    ('Markdown', '3.2.2', {
```

Comment:
@terjekv These should be revisited, some may already included with `Python`, some should probably be updated?

Reply:
What I did: `eb --dump-env` on this EC, then source it and run the linked script. That should give you a nice list of packages and versions to use.